Modules/posixmodule.c

```diff
@@ -677,6 +677,14 @@ PyOS_AfterFork_Parent(void)
     run_at_forkers(interp->after_forkers_parent, 0);
 }
 
+void
+PyRemoteDebugCall_Reset(PyThreadState *tstate)
```

Comment:
```suggestion
static void
PyRemoteDebugCall_Reset(PyThreadState *tstate)
```

Reply:
There is no need to make this call look like an exported function since is used within the same compilation unit